### PR TITLE
oneskyapp-pull-android-translations 0.0.2

### PR DESCRIPTION
**Pull request by prcaen**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by prcaen** [comment]
Hi,

What's the next step?

Thanks

File: steps/oneskyapp-pull-android-translations/0.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/oneskyapp-pull-android-translations/0.0.2/step.yml]: git dependency is considered redundant: https://discuss.bitrise.io/t/golang-as-a-dependency/6989/8?u=koral

File: steps/oneskyapp-pull-android-translations/0.0.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+source_code_url: https://github.com/prcaen/bitrise-step-oneskyapp-pull-translations
+support_url: https://github.com/prcaen/bitrise-step-oneskyapp-pull-translations/issues
+published_at: 2018-12-18T10:23:01.507167+01:00
+source:
+  git: https://github.com/prcaen/bitrise-step-oneskyapp-android-pull-translations.git
+  commit: 688ce65ac6349a739b4d9ffc197eddbcc9b9ae78
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
```
Comment (BirmacherAkos) [inline steps/oneskyapp-pull-android-translations/0.0.2/step.yml]: Does this step work only for Android?
If so, you should specify the project_type_tags in the step.yml:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention

**Comment by BirmacherAkos** [comment]
Hi @prcaen !

Thanks for the step 🥇 
Could you please check my comments?

**Comment by BirmacherAkos** [comment]
Hi @prcaen !

I will close this PR because publishing the changes will require another step release (with new patch version), which will create a new PR anyway.

Let me know if there is any uncertainty regarding my previous comment.

**Pull request closed without merging**